### ISSUE TITLE
Unable to load in production build

### ISSUE THREAD

**Issue by josx** [tkambler/electron-preferences#119]
After install with `npm install electron-preferences`, and test it on devel it is working pretty well. But once i want to make a production build i dont know how to include electron-preferences build files. BTW I am using [ERB](https://github.com/electron-react-boilerplate/).

The example is including just lib files, and not using the npm package. So I dont know how to continue.

Errors:
```
unable to load preload script: app/preload.js
```
```
Not allowed to load local resource: app/build/index.html
```
Maybe It is not on the scope of this project but would be great any clue to solve it.
It seems that I have to include the build path of the library in some way on my build (but the rare thing is that is working pretty web in dev env).

**Machine:**
 - OS: Debian Gnu/Linux
 - Electron version: 8.x
 - electron-preferences version: 2.x.x

**Comment by pvrobays** [comment]
Hi @josx 

I'm not familiar with ERB, but I assume in production it'll build all your JS to a single file and serve that to electron?

While building I assume it'll scan all the imports and add electron-preferences as well, but only the API. Not the asset files for the actual preference window. You'll probably have to copy the asset files yourself as a build step of your production app. The files you should copy are:
- `preload.js`
- everything from the `build` directory

I'm currently swamped, but if at a later stage I have the time, I'll check out ERB myself and might try to find a better solution. If you have any feedback or suggestions, feel free to add them here.

**Comment by josx** [comment]
Thanks a lot for replying. I will test your proposal to copy files and send you feedback.

**Comment by pvrobays** [comment]
Hi @josx 

I'm curious if you've found the time to get everything working with ERB?

**Comment by josx** [comment]
So sorry but i couldnt, but as you said i can make an script to copy those file before building.

**Comment by lacymorrow** [comment]
I believe I have a fix for this incoming

**Comment by pvrobays** [comment]
Fix released in 2.6.0
